src/wwwroot/template-editor/adapterService.js

```diff
@@ -0,0 +1,356 @@
+(function() {
+    'use strict';
+  
+    angular
+      .module('mseditor')
+      .service('adapterService', adapterService);
+  
+      adapterService.$inject = [
+      '$http',
+      '$q'
+    ];
+  
+    function adapterService($http, $q) {
+      var service = {
+        getCampaign: getCampaign,
+        campaignSaveChanges: campaignSaveChanges,
+        campaignSaveChangesAsTemplate: campaignSaveChangesAsTemplate,
+        getCampaignRssPreview: getCampaignRssPreview,
+        deleteImages: deleteImages,
+        getImagesForCampaign: getImagesForCampaign,
+        getPreviewImage: getPreviewImage,
+        getPreviewImageWithPlay: getPreviewImageWithPlay,
+        getTiendaNubeProducts: getTiendaNubeProducts,
+        getSettings: getSettings
+      };
+  
+      return service;
+
+      function getPreviewImageWithPlay(params) {
+        /**
+         * Return a preview placeholder image
+         * @method
+         * @argument {object} params - Example:
+         * {
+         *  fileName: fileName,
+         *  providerName: providerName,
+         *  idCampaign: campaign.getIdCampaign(),
+         *  idTemplate: campaign.getIdTemplate()
+         * }
+         */
```

Comment:
If I am not wrong, the comments should [be placed immediately before the function declaration](http://usejsdoc.org/about-getting-started.html), Am I missing something?